starter_code/queries.md

```diff
@@ -5,45 +5,89 @@
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
 <!-- Your Code Goes Here -->
-
+query: {name: "Babelgum"}
+projection: {name:1, _id:0}
+sort: 
+skip: 
+limit:
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
 <!-- Your Code Goes Here -->
-
+query: {number_of_employees: {$gt:5000}}
+projection:
+sort: {number_of_employees:1} 
+skip:
+limit: 20
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
 <!-- Your Code Goes Here -->
-
+query: {$and:[{founded_year: {$gte: 2000}}, {founded_year: {$lte: 2005}} ] }
```

Comment:
That is one way, another short way would be nesting the limits on founded year in itself like this:

{founded_year: {$gte : 2000, $lte: 2005}}

Reply:
Yeah! I was thinking, why it is so long:) Thanks :)